packages/kbn-i18n/README.md

```diff
@@ -175,7 +175,7 @@ export function getGreetingMessage(userName) {
   return i18n.translate('hello.wonderful.world', {
     defaultMessage: 'Greetings, {name}!',
     values: { name: userName },
-    context: 'This is greeting message for main screen.'
```

Comment:
I think that property has been renamed over time, and it's now called `description`.

Reply:
right, thank you for fixing it